### PR TITLE
Update/fix goreleaser config

### PR DESCRIPTION
**Pull request by lecaros**

### CODE REVIEW

**Review by cosmo0920 — APPROVED** [review]
I confirmed that `goreleaser build --snapshot` works.

File: .goreleaser.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
```
Comment (cosmo0920) [inline .goreleaser.yaml:26]: This is just nitpick and no needed to implement this PR but we need to add arm64 target for linux and darwin and perhaps for Windows?

Comment (cosmo0920) [reply .goreleaser.yaml:26]: FYI: Windows ARM64, Apple Silicon macOS, and Linux AArch64 are also valid targets.

Comment (lecaros) [reply .goreleaser.yaml:26]: I'll look into those targets to add a new PR. Thanks @cosmo0920!

**Pull request merged**